src/db/postgres.ts

```diff
@@ -6,10 +6,10 @@ import { Game, gameAttributes } from "./entities/game";
 export class PostgresStore extends Async {
   private sequelize: Sequelize;
 
-  constructor(uriOrOptions: Options | string) {
+  constructor(uriOrOptions: Options | string, extraOptions?: Options) {
```

Comment:
Let's narrow down the types to the valid cases.

```suggestion
  constructor(uri: string, options?: Options);
  constructor(options: Options);
  constructor(uriOrOptions: Options | string, extraOptions?: Options) {
```